gradle/src/main/groovy/io/sentry/android/gradle/SentryPluginExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package io.sentry.android.gradle
+
+class SentryPluginExtension {
```

Comment:
I see this being referenced but I don't understand its use, could we add an explanatory comment?